lib/Doctrine/Common/Persistence/AbstractManagerRegistry.php

```diff
@@ -189,7 +191,7 @@ public function getManager($name = null)
     public function getManagerForClass($class)
     {
         // Check for namespace alias
-        if (strpos($class, ':') !== false) {
```

Comment:
We should really just deprecate class aliases: the `::class` meta-constant provided by PHP since PHP 5.5 is sufficient in most scenarios.

Reply:
That could work also. Should that be the focus now?

Reply:
I think so - the aliasing was really a pre-PHP-5.5 thing, because typing full class names was quite annoying.

Reply:
That would then be out of the scope of this issue and PR?